templates/web-deployment.yaml

```diff
@@ -96,6 +96,18 @@ spec:
             - name: CONCOURSE_COMPONENT_RUNNER_INTERVAL
               value: {{ .Values.concourse.web.componentRunnerInterval | quote }}
             {{- end }}
+            {{ - if .Values.concourse.web.defaultGetTimeout }}
```

Comment:
@evanchaoli Thanks for the PR. Could you remove the space between the all the `{` and `-`? The lint test is failing because of it.

e.g. ``{{- if .Values.concourse.web.defaultGetTimeout}}``